app/Helpers/PrecinctImporter.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+namespace App\Helpers;
+use Akeneo\Component\SpreadsheetParser\SpreadsheetParser;
+use App\City;
+use App\Precinct;
+use Illuminate\Support\Facades\File;
+
+class PrecinctImporter
+{
+
+    public function importFromFile(\Symfony\Component\HttpFoundation\File\File $file, bool $deleteFileAfter)
```

Comment:
import the class `\Symfony\Component\HttpFoundation\File\File` and use the simplified FQN (`File`) (for keeping the usages of type hints as in rest of the project)

Reply:
there is a name collision between `Illuminate\Support\Facades\File` and `\Symfony\Component\HttpFoundation\File\File`. I can alias the Facades one to Filesystem ( to have something like Filesystem::delete() and use the simplified File for type hinting.

Reply:
Decided to move away from framework specific classes for file and used SplFileObject